module/zfs/dmu_send.c

```diff
@@ -1942,6 +1943,14 @@ setup_featureflags(struct dmu_send_params *dspp, objset_t *os,
 		*featureflags |= DMU_BACKUP_FEATURE_LZ4;
 	}
 
+	/* XXX: Allan: should be able to use embedok without implying ZSTD */
```

Comment:
Is it clear what this means?
If not: remove it
If it's clear, is it valid/required etc. ?
if not: remove it.

Reply:
i dont know what he means

Reply:
In that case: DELETE.